### ISSUE TITLE
Error with periph sections

### ISSUE THREAD

**Issue by mnjames** [sillsdev/ptx2pdf#868]
I'm getting this error:
```
! Undefined control sequence.
\MS:zgetperiph ...me \ifcsname keepperiph-\attr@b 
                  \endcsname \keeptriggertru...
\st@ndalonemilestone ...\thismil@stone \endcsname 
                  \fi \ifx \attr@b \relax \c...
\processmil@stone ... \empty \st@ndalonemilestone 
                  \else \if \milestoneOp s\r...
\*...tr@b \fi \mst@nestylefalse \processmil@stone 
                  \else {*}\fi 
\@ttrSl@sh ...mpty \else \x@ \@@ttrSlash #1\relax 
                  \relax \fi 
l.55 \zgetperiph|intepistles\*
------------------------------------------------------------------------------------------
This might be related to a USFM marker error (using an unsupported marker).
Try running 'Basic Checks' in Paratext to validate markers.

PTXprint Version 2.3.10-g9d21966c
```

Note that it's _very_ possible that I've simply set things up incorrectly. This is the first time I'm trying to use this feature.

My `INT` book in PT has this in it, with introductory material following it:
```
\periph Epistles Introduction|id="intepistles"
```
My Front Matter section in PTXprint has this:
```
\rem ----------Page 4----------------
\zgetperiph|intepistles\*
```
And I've checked the box in the Body tab which says `Use Periphery Introductory Sections`.

**Comment by davidg-sil** [comment]
Can you paste the relevant bit of the logfile for that error, please. At the moment I'm guessing that the python code has mangled the line breaks. I hope.... Otherwise it's saying that the  2nd time it wanted to `\attr@b`  it wasn't defined defined any more, but it was fine the first time..?   
Oh... maybe it's saying that `\attr@b` gets undefined by your periphery.  I suppose that's possible. If that's the bug then the push I've just done ought to fix it.

**Comment by mnjames** [comment]
I tried out your fix, and things worked fine.

Then, because I couldn't remember exactly what the problem looked like, I went back to a previous github commit and tested with it... and it worked fine.
So, things are working fine, and it's possible that your commit wasn't necessary. I can't think of a single thing I've changed, so I have absolutely no idea why it wasn't working 3 days ago and is now (both with or without your commit).
